packages/compiler/config/config-interpolation.ts

```diff
@@ -1,7 +1,12 @@
 import { createDiagnosticCollector, ignoreDiagnostics } from "../core/diagnostics.js";
```

Comment:
can't comment on renamed file without changes (just above) so commenting here, but cmd/typespec.js should be cmd/tsp.js to match the command name.